requests-for-comments/2024-06-21-warehouse-launch-plan.md

```diff
@@ -0,0 +1,50 @@
+## [Marketing] Data warehouse launch plan
+
+The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 
+
+### Ahead of launch
+
+- [x] Publish a [Stripe data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8662) - Marketing
+- [x] Publish a [Hubspot data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8664) - Marketing
+- [x] Publish a [Zendesk data warehouse tutorial](https://github.com/PostHog/posthog.com/pull/8858) - Marketing
+- [ ] Create dashboard templates for 2x common usecases - @ivanagas
+- [x] Update docs as needed - Marketing
+- [ ] Add Data Warehouse to in-app 'Explore the docs' menu - https://github.com/PostHog/posthog/pull/22715
+- [x] Key art - @lottiecoxon (we have some already, open to new though)
+
+
+### Launch day activity
+
+- [ ] Publish a launch blog piece about why data warehouse is important - @joethreepwood 
+- [ ] Update pricing page with data warehouse pricing - Growth
+- [ ] Update in-app onboarding with data warehouse option - Growth
```

Comment:
@raquelmsmith I assume we're doing this. Can you let me know when it's live/when the PR is ready? I'll factor it into an onboarding email flow too.

Reply:
For now I'm just going to put a button in the initial /products page that points them to the link sources page.

Reply:
```suggestion
- [x] Update in-app onboarding with data warehouse option - Growth
```